CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 **Features**:
 
 - Add `sentry_clear_attachments()` to allow clearing all previously added attachments in the global scope. ([#1290](https://github.com/getsentry/sentry-native/pull/1290))
+- Compiles also on Xbox One ([#1294](https://github.com/getsentry/sentry-native/pull/1294))
```

Comment:
The PR says: "feat: xbox one support" but the changelog only mentions compilation. Do you expect another change before Xbox One can be considered supported? I think it should be made clear from the changelog.

Reply:
Good point, I made it clear that it's about compilation for now since I can't validate at this point the support. But we'll order a devkit and get this tested at some point and add to docs and communicate things better then.
Thank you!